tests/post/features/solutions.feature

```diff
@@ -0,0 +1,30 @@
+@local @solution
+Feature: Solutions
+    Scenario: Deploy Solution
+        Given the Kubernetes API is available
+        And no Solution 'example-solution' is imported
+        And no Solution environment 'example-environment' is available
+        And the Solution Configuration file is absent
+        When we import a Solution archive '/var/tmp/example-solution.iso'
+        Then Solution archive 'example-solution' is imported correctly
+        And Solution 'example-solution' version '0.1.0-dev' is available
+        When we activate Solution 'example-solution' version '0.1.0-dev'
+        Then Solution 'example-solution' version '0.1.0-dev' is activated
+        And CRD 'versionservers.example-solution.metalk8s.scality.com' exists in Kubernetes API
+        And CRD 'clockservers.example-solution.metalk8s.scality.com' exists in Kubernetes API
+        When we create a solution environment 'example-environment'
+        Then solution environment 'example-environment' is available
+        When we remove Taints on node 'bootstrap' before deployment
+        And we deploy Solution 'example-solution' in environment 'example-environment' with version '0.1.0-dev'
+        Then the 'pods' list should not be empty in the 'example-environment' namespace
+        And we have '1' available 'status.available_replicas' for Solution 'example-solution-operator' Deployment in environment 'example-environment'
+        And we have '1' available 'status.available_replicas' for Solution 'example-solution-ui' Deployment in environment 'example-environment'
```

Comment:
You could have used what's already done for the rest of the tests (defined in conftest.py).
```
we have 1 running pod labeled 'name=example-solution-operator' in namespace 'example-environment'
we have 1 running pod labeled 'app=example-solution-ui' in namespace 'example-environment'
```